### ISSUE TITLE
Calendario + day display

### ISSUE THREAD

**Issue by kusumj** [codrops/Calendario#43]
Hello,

I need to display the day and event as below : 
'24-08-2015' : '<a href="#" >Event 1</a>',
'24-08-2015' : '<a href="#">Event 2</a>',

can someone please help me on the above.

Thanks

**Comment by deviprsd21** [comment]
Display where?

**Comment by kusj** [comment]
Hello ,

I need to render the code as follows : 

var codropsEvents = {
'07-08-2015' : '<a href="#" class="concert">Event 1t</a>',
'07-08-2015' : '<a href="#" class="concert">Event 2t</a>',
};

The date is dynamic and I am unable to group the event under similar date.
That is why i need to render the code as above. But it is talking only the last one that is event 2.

Thanks for your help.

**Comment by deviprsd21** [comment]
Yea you can do that, but it would be good if you first use the v4 of calendario available here, https://github.com/deviprsd21/Calendario. Then, you can see how this one is written, https://github.com/deviprsd21/Calendario/blob/master/js/dataTimeline.js

**Comment by kusj** [comment]
Hello ,

As advised I have used v4 of calendario. But my issue is still the same.

I want to be able to render the date and content as below : 

'09-03-2015' : [{content : 'Graduation Exams',  allDay: true}],
'09-03-2015' : [{content : 'asasas',  allDay: true}]

Is it possible with calendario ?

Thanks in advance,

kusum

**Comment by deviprsd21** [comment]
Yes it is, you just didn't read the code carefully, never mind. Write it like this
```
'09-03-2015' : [{content : 'Graduation Exams', allDay: true}, {content : 'asasas', allDay: true}]
```
You see the `[ ]` are arrays which contain the event objects and `{ }` are objects which have the event details, so to add more just `[{ }, { }, { }, so on]`

**Comment by blacRose** [comment]
Why can we not add two day events separately like his first post and have it draw both?

**Comment by deviprsd21** [comment]
@blacRose Because this is in a JavaScript Object format, if you try to do like the first post, the second line will override the first one. Think of it as an array, and the date is the index key, so the same key will replace it. I hope I'm clear enough?

**Comment by deviprsd21** [comment]
Also why don't you try building hash objects similar to this in ruby, and then converting them to json, it will as it should work. The `[ ]` are arrays, and `{ }` objects
